### DISCUSSION TITLE
Change Password Requirements

### DISCUSSION THREAD

**Discussion by bmosterhaus** [plausible/analytics#3327] [Self-Hosted Support] [answered]
Currently running a self-hosted container of plausible.  Is there a way to change the minimum password requirements from 6 to 14 characters?

**Comment by ruslandoga** [comment, marked as answer]
👋 @bmosterhaus

The minimum password length is hard-coded: https://github.com/plausible/analytics/blob/ae0019afe50e84e8ed02c6db9dc8d96fee04c4a0/lib/plausible/auth/user.ex#L46

**Reply by ruslandoga** [reply]
The only way to make it 14 right now that I can think of is to change it in the source code and rebuild the container image.
